Add contact form

jips-on opened the pull request:
**Built contact form for the portfolio website** 

- Build with HTMl,CSS
- Framework used-grid, flexbox
- Add Linters for HTML and CSS file

jips-on committed:
Add contact form

Add contact form

DiamondNath07 committed:
add minlength to username field

jips-on committed:
Updated html file and css file

Update the HTML file by adding the form URL and Update the CSS file by styling the contact form.

jips-on committed:
Merge branch 'Add-contact-form' of https://github.com/jips-on/Portfolio-Mobile-version into Add-contact-form

jips-on committed:
Fix errors

Fix errors

topeogunleye commented on index.html:
```diff
@@ -166,5 +216,33 @@ <h3>LET'S CONNECT</h3>
+            <input type="email" name="email" id="email" placeholder="natajayanj14@gmail.com" required>
+          </li>
+          <li>
+            <textarea name="user_message" id="msg" placeholder="Write your message here" maxlength="300" required></textarea>
```
- [ ] As written in the project requirements. I kindly suggest that you increase your text area maxlength to 500 because the Message field should accepts up to 500 chars

topeogunleye commented on index.html:
```diff
@@ -166,5 +216,33 @@ <h3>LET'S CONNECT</h3>
-</html>
\ No newline at end of file
+<footer>
+    <form action="https://formspree.io/f/xrgdkyvk" method="post">
```
## I kindly suggest that you sticks to the design templates in Figma as much as possible, this includes the font, colours, images and spacing (margin/padding) by:
- [ ] Please ensure your skills and contact section do not overlap because in the figma design there is a big space between them.
- [ ] Please add the border radius to the top left side of the contact form as seen in the Figma design.

| Your Design  | Figma Design |
| ------------- | ------------- |
| <img width="220" alt="image" src="https://user-images.githubusercontent.com/52934128/193851889-2f602be0-bb49-4fcf-92ab-6a05e510d0ce.png"> | ![image](https://user-images.githubusercontent.com/52934128/193855599-b354216a-b872-46eb-8ec0-2416d535f75b.png) |

topeogunleye commented on style.css:
```diff
@@ -446,3 +467,93 @@ hr {
+  font-size: 17px;
+}
+
+input[value] {
```
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```

topeogunleye reviewed (changes requested):
Hi Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![Try-Again](https://user-images.githubusercontent.com/52934128/180646051-aed7efa6-6d15-4853-9962-73186e366f2c.gif)

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your @topeogunleye so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jips-on committed:
Make changes suggested code reviewer

Updated background image by changing position and enabled the button interaction.

jips-on committed:
Fix linters

youmari commented on index.html:
```diff
@@ -166,5 +166,33 @@ <h3>LET'S CONNECT</h3>
+            project that needs coding. I’d <br>
+            to help with it
+          </p>
+        </header>
+        <ul style="list-style: none">
+          <li>
+            <input type="text" name="user-name" id="name" minlength="5" placeholder="Yeremias" required maxlength="30">
+          </li>
+          <li>
+            <input type="email" name="email" id="email" placeholder="natajayanj14@gmail.com" required>
+          </li>
+          <li>
+            <textarea name="user_message" id="msg" placeholder="Write your message here" maxlength="500" required></textarea>
+          </li>
+          <li>
+            <input type="submit" name="submit" value="Get in touch">
+          </li>
```
- [ ] [OPTIONAL] it would be better if you remove the black border from the inputs and also add the right padding to them in order to match The Figma design 
- Yours 
<img width="359" alt="Screen Shot 2022-10-04 at 17 32 12" src="https://user-images.githubusercontent.com/61790587/193875203-c6479052-d5ad-4196-9264-d5a517e7ad69.png">

youmari reviewed (approved):
# Project Approved :trophy: :tada: 🟢 

Hi @jips-on 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).
